dist/index.html

```diff
@@ -53,6 +53,19 @@ <h3>Your Trips</h3>
     </div>
   </section>
   <main id="tripCardsContainer">
+    <h4 class="login-title" id="loginTitle">Login To Continue</h4>
+    <form class="login-form" id="loginForm">
+      <div class="login-info-container">
+        <label for="userName">Username:</label>
+        <input class="login-input" type="text" id="userName" name="username" placeholder="Username" aria-required="true">
+      </div>
+      <div class="login-info-container">
+        <label for="password">Password:</label>
+        <input class="login-input" type="password" id="password" name="password" placeholder="Password" aria-required="true">
+      </div>
+      <button class="login-button" id="loginButton">LOGIN</button>
+      <p class="login-error-message hidden" id="loginErrorMessage">Please Fill Out All Sections!</p>
+    </form>
   </main>
   <!-- <h4 class="no-trips-message hidden" id="noTripsMessage">You Have No Pending Trips. Time To Book!</h4> -->
```

Comment:
I may be missing something but you probably want to delete your commented out code on lines 70 - 72